install.php

```diff
@@ -109,19 +109,24 @@
 
 	if (empty($message))
 	{
-		//Create config
+		$fullname = $_POST['surname'] . " " . $_POST['name'];
+		//Create config & privacy policy
 		$config = file_get_contents("config.php.template");
+		$policy = file_get_contents("policy.php");
 		$config = str_replace("##name##", $_POST['servername'], $config);
 		$config = str_replace("##title##", $_POST['title'], $config);
 		$config = str_replace("##url##", $_POST['url'], $config);
-		$config = str_replace("##mailer##", $_POST['mailer'], $config);
```

Comment:
Does this break anything? Seems to me it would prevent mailer to have the name you set in installation script.

Reply:
Oh true, I didn't want to remove it, will add it again.